Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -297,6 +331,18 @@
         "VirtualApplicationDoesNotExist": "Virtual application doesn't exists : %s",
         "JSONParseError": "Unable to parse JSON file: %s. Error: %s",
         "JSONvariablesubstitutionappliedsuccessfully": "JSON variable substitution applied successfully.",
-        "XMLvariablesubstitutionappliedsuccessfully": "XML variable substitution applied successfully."
+        "XMLvariablesubstitutionappliedsuccessfully": "XML variable substitution applied successfully.",
+        "failedtoUploadFileToKudu": "Unable to upload file: %s to Kudu (%s). Status Code: %s (%s)",
+        "failedtoUploadFileToKuduError": "Unable to upload file: %s to Kudu (%s). Error: %s",
+        "ExecuteScriptOnKudu": "Executing given Script on Kudu: %s",
+        "FailedToRunScriptOnKuduError": "Unable to run the script on Kudu: %s. Error: %s",
+        "FailedToRunScriptOnKudu": "Unable to run the script on Kudu: %s. Status Code: %s (%s)",
+        "SciptExecutionOnKuduSuccess": "Successfully executed script on Kudu.",
+        "SciptExecutionOnKuduFailed": "Executed Script returned '%s' as return code. Error: %s",
+        "FailedtoDeleteFileFromKudu": "Unable to delete file: %s from Kudu (%s). Status Code: %s (%s)",
+        "FailedtoDeleteFileFromKuduError": "Unable to delete file: %s from Kudu (%s). Error: %s",
+        "ScriptFileNotFound": "Script file '%s' not found.",
+        "InvalidScriptFile": "Invalid Script file '%s' provided. Valid extensions are .bat and .cmd",
+        "RetryForTimeoutIssue": "Script execution failed with timeout issue. Retrying once again."
```

Comment:
@kmkumaran , will get the app offline content, error message and help string reviewed by @azooinmyluggage